[#37] 포포 컨셉 카테고리 설정 구현

hyun99999 opened the pull request:
### Descriptions
- PopoConcept 폴더 생성(포포 컨셉 생성 목적)
- Login 폴더 생성
- Category 뷰 구현
- backButton 이미지 추가

- Moya pod install
- UIView+Extension 에 makeShadow() 메서드 추가
- 온보딩 문구 수정
- APIModel 폴더에 서버통신을 위한 세팅(GenericResponse, NetworkResult)
- ConceptViewController 에서 쓰기위한 모델 세팅(ConceptModel)

- baseURL 상수 설정(GeneralAPI)
- HTTP 통신 가능하도록 Info.plist 수정
- ConceptModel -> ConceptData 클래스 이름변경
- Popo-API 에 해당하는 service 파일 생성(포포리스트조회하는 case 만 완성)

_개발을 뭔가.. 중구난방으로 한 느낌인데 다음에는... 이슈대로 할게요 ㅜ_

hyun99999 committed:
Merge branch 'develop' of https://github.com/hyun99999/POPO-iOS into feature/#31

hyun99999 committed:
[#37] 포포 컨셉 카테고리 설정 구현

### Descriptions
- PopoConcept 폴더 생성(포포 컨셉 생성 목적)
- Login 폴더 생성
- Category 뷰 구현
- backButton 이미지 추가

### Related Issue
#37

hyun99999 committed:
Merge branch 'develop' of https://github.com/hyun99999/POPO-iOS into feature/#37

hyun99999 committed:
[#37] 포포 컨셉 카테고리 설정 구현

### Descriptions
- Moya pod install
- UIView+Extension 에 makeShadow() 메서드 추가
- 온보딩 문구 수정
- APIModel 폴더에 서버통신을 위한 세팅(GenericResponse, NetworkResult)
- ConceptViewController 에서 쓰기위한 모델 세팅(ConceptModel)

### Related Issue
#37

hyun99999 committed:
[#37] 포포 컨셉 카테고리 설정 구현

### Descriptions
- baseURL 상수 설정(GeneralAPI)
- HTTP 통신 가능하도록 Info.plist 수정
- ConceptModel -> ConceptData 클래스 이름변경
- Popo-API 에 해당하는 service 파일 생성(포포리스트조회하는 case 만 완성)

### Related Issue
close #37

jeongchoyi commented on popo/popo/Sources/Cell/Onboarding/ConceptSelectCell.swift:
```diff
@@ -9,6 +9,8 @@ import UIKit
 
 class ConceptSelectCell: UICollectionViewCell {
 
+    static let identifier = "ConceptSelectCell"
+    
```
요런거 Const에 Xib Struct 만들어서 한번에 관리하는건 어떤가여?
ViewController랑 Storyboard 한 것처럼요!
![image](https://user-images.githubusercontent.com/28949235/134647953-23d69692-1443-4e0b-99a4-2759d6e60cfe.png)
요런식..

hyun99999 committed:
[#37] 포포 컨셉 카테고리 설정 구현

### Descriptions
- 코드리뷰적용(Const 에 Xib 파일에서 identifier 관리)

### Related Issue
close #37
